src/tests/networking_tests.rs

```diff
@@ -720,7 +745,7 @@ pub mod net_tests {
             println!("RECV");
```

Comment:
Can get rid of this if the tests are working now -- this was just for me to check when deadlocks were happening.

Reply:
The same thing on line 736